src/proxy_lib/proxy_lib.c

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+*/
+
+// UMF proxy library
```

Comment:
can you extend this description a little - what is a proxy lib?

Reply:
Done